nvdaAPIVersions.json

```diff
@@ -0,0 +1,327 @@
+[
```

Comment:
i don't think we should host a copy of this file here.
Instead, the dependency should be passed in via command line e.g. `runvalidate <pathToAddonMetadataFile.json> <pathTonvdaAPIVersions.json>`